api/managementpb/backup/restores.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+
+package backup.v1beta1;
+
+option go_package = "api/managementpb/backup;backupv1beta1";
+
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+import "managementpb/backup/common.proto";
+
+// RestoreHistoryItem represents single backup restore item.
+message RestoreHistoryItem {
```

Comment:
Maybe add artifact_id, so we know from which backup it was restore. At least in DB.

Reply:
added